src/view/reviewManager.ts

```diff
@@ -649,6 +649,57 @@ export class ReviewManager implements vscode.DecorationProvider {
 		return selected;
 	}
 
+	private async getPullRequestTitleAndDescriptionDefaults(progress: vscode.Progress<{message?: string, increment?: number}>): Promise<{ title: string, description: string } | undefined> {
+		progress.report({ increment: 5, message: 'Looking for pull request template...' });
+		const pullRequestTemplates = await this._prManager.getPullRequestTemplates();
+		let template: vscode.Uri | undefined;
+
+		if (pullRequestTemplates.length === 0) {
+			progress.report({ increment: 5, message: 'No pull request template found. Creating pull request...' });
```

Comment:
I'm wondering if this should be transparent to users who are not aware of Pull Request Templates.